core/mm/core_mmu.c

```diff
@@ -373,6 +418,11 @@ void core_mmu_set_discovered_nsec_ddr(struct core_mmu_phys_mem *start,
 	 */
 
 #ifdef CFG_SECURE_DATA_PATH
+	if (dtb_get_sdp_region()) {
+		DMSG("region base %p %zu", (void *)sec_sdp.paddr, sec_sdp.size);
```

Comment:
prefer without a cast
```c
		DMSG("region base %#"PRIxPA" %zu", sec_sdp.paddr, sec_sdp.size);
```
I don't think it's the best place to print existence of the area.
It would be better either where found, from `dtb_get_sdp_region()` or no specific `DMSG()`.

Reply:
ok let's remove this log.